Library/Formula/ucspi-tools.rb

```diff
@@ -0,0 +1,19 @@
+class UcspiTools < Formula
+  homepage "https://github.com/younix/ucspi/blob/master/README.md"
+  url "https://github.com/younix/ucspi/archive/7c59e7649cbce470452116595b5c00a26840d4cf.tar.gz"
```

Comment:
With the latest release the mountain lion test fails, see test results from commits above.
So I fixed it with a new commit: https://github.com/younix/ucspi/commit/7c59e7649cbce470452116595b5c00a26840d4cf